### ISSUE TITLE
Add Manifest to compiled project

### ISSUE THREAD

**Issue by BlackRouter** [Nuitka/Nuitka#744]
```
0.6.8.3
Python: 3.7.7 (tags/v3.7.7:d7c567b08f, Mar 10 2020, 09:44:33) [MSC v.1900 32 bit (Intel)]
Executable: C:\Users\Moein\AppData\Local\Programs\Python\Python37-32\python.exe
OS: Windows
Arch: x86
```

how to add manifest run project as administrator in windows?

**Comment by kayhayen** [comment]
Stupid question, what do you mean with manifest? Can you point to an example of those?

**Comment by BlackRouter** [comment]
@kayhayen in pyinstaller have switch `--uac-admin` and add manifest for run as administrator exe file , can you add this switch in nuitka?

**Comment by kayhayen** [comment]
Nuitka has gained the option `--windows-uac-admin` and `--windows-uac-uiaccess` on the factory branch. The change the manifests as needed.

**Comment by kayhayen** [comment]
Released as part of 0.6.10
